static/js/components/Loading_test.js

```diff
@@ -1,134 +1,44 @@
 // @flow
 import React from "react"
 import { assert } from "chai"
-import { shallow } from "enzyme"
+import { mount } from "enzyme"
 
-import Loading from "./Loading"
+import withLoading from "./Loading"
+
+class Content extends React.Component {
+  render() {
+    return <div>CONTENT</div>
+  }
+}
+const LoadingContent = withLoading(Content, () => true, () => {})
 
 describe("Loading", () => {
-  const renderLoading = restStates =>
-    shallow(<Loading restStates={restStates} renderContents={() => <div>CONTENT</div>} />)
+  const renderLoading = (loaded: boolean, errored: boolean) => {
+    return mount(<LoadingContent loaded={loaded} errored={errored} />)
+  }
 
   describe("should render loading correctly", () => {
```

Comment:
if you wouldn't mind I think we should just have `it` blocks here, I don't think we have so many tests that we need nested `describe` blocks